target/snitch_cluster/sw/apps/snax-mac-tiled/src/snax-mac-tiled.c

```diff
@@ -0,0 +1,147 @@
+// Copyright 2023 KU Leuven
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+//
+// Josse Van Delm <jvandelm@esat.kuleuven.be>
+// Ryan Antonio <rgantonio@esat.kuleuven.be>
+
+#include "snrt.h"
+
+#include "data.h"
+
+void snax_mac_launch() {
+    // Write start CSR to launch accelerator
+    write_csr(0x3c0, 0);
+}
+
+void snax_mac_sw_clear() {
+    // poll csr 0x3c3 until HWPE MAC accelerator is finished
+    write_csr(0x3c5, 0);
```

Comment:
0x3c3 or 0x3c5?

Reply:
hehe incorrect comment it should've not been there lol